src/libs/highlevel/README.md

```diff
@@ -142,7 +142,31 @@ Errors which do not originate inside the high-level API itself are wrapped into
 `ELEKTRA_ERROR_CODE_LOW_LEVEL`. The high-level Error API provides methods (`elektraKDBError*`) to access the properties of the low-level
 error. You can also access the key to which the error was originally attached, as well as any possible low-level warnings.
 
-TODO: explanation of low-level code or removal of this feature for this release?
+To get the original low-level error code, description, severity, group, module and reason you can use these functions:
```

Comment:
@kodebach Is this a good design to split the accessors functions of low-level errors and the other errors? I suppose people also want to get the errorcode, severity, ... for high-level error codes?

Reply:
You are right, `elektraErrorCode` and `elektraErrorSeverity` exist too, but only `elektraErrorDescription` is mentioned in the README.md

Reply:
As said above, I think we should now publish a minimal API. We can extend it later easily.